### PR TITLE
What is data structure

### PR DESCRIPTION
**Pull request by Hanuvesh**

### CODE REVIEW

File: what is data structure.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+## What is dsa?
+## Classification of Data Structure
+## Popular types of Data Structures:
+
+
+
+# What is dsa?
+
+A data structure is a storage that is used to store and organize data. It is a way of arranging data on a computer so that it can be accessed and updated efficiently.
```
Comment (Nexengineer) [inline what is data structure.md:9]: is a way to manage data

**Comment by harshil-1402** [comment]
Hiii, so we need to make some changes in the file what is data structures.md right?

**Comment by Nexengineer** [comment]
yes

**Comment by Nexengineer** [comment]
you have to rename the file

**Comment by Hanuvesh** [comment]
what changes you want ? what to rename the file

File: what is data structure.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+
+# Classification of Data Structure
+
+* Linear data structure: Data structure in which data elements are arranged sequentially or linearly, where each element is attached to its previous and next adjacent elements, is called a linear data structure. 
+
+Examples of linear data structures are array, stack, queue, linked list, etc.
+
+* Static data structure: Static data structure has a fixed memory size. It is easier to access the elements in a static data structure.
+
```
Comment (harshil-1402) [inline what is data structure.md:21]: The contents of static data structures can be modified without changing the memory allocation of these data structures

File: what is data structure.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+An example of this data structure is an array.
+
```
Comment (harshil-1402) [inline what is data structure.md:23]: For example in an array you change the element at a particular location/index but we did not increase/decrease the size of the array, that is, the memory allocation for the array is the same.

File: what is data structure.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+* Dynamic data structure: In dynamic data structure, the size is not fixed. It can be randomly updated during the runtime which may be considered efficient concerning the memory (space) complexity of the code. 
+
```
Comment (harshil-1402) [inline what is data structure.md:25]: Dynamic data structures are designed to give the liberty of change of data structures in the run time.

**Review by harshil-1402 — CHANGES_REQUESTED** [review]

**Comment by harshil-1402** [comment]
I have submitted the review proposing some changes

**Pull request closed without merging**